mkdocs.yml

```diff
@@ -103,6 +103,7 @@ nav:
           - Link to Code Source: guides/advanced/link-to-code-source.md
           - Testing: guides/advanced/testing.md
           - Backfill: guides/advanced/backfill.md
+          - Using Environments: guides/advanced/using-environments.md
```

Comment:
I think this should go on the top, it's going to be a very common use case.

Reply:
updated